package-lock.json

```diff
@@ -0,0 +1,16355 @@
+{
+  "name": "project",
+  "version": "0.1.0",
+  "lockfileVersion": 1,
```

Comment:
this project uses yarn...refer readme.md

Reply:
We are also using yarn .

Reply:
this file is only generated when we use npm to install the dependencies